clients/iota-go/iotaclient/iotaclienttest/api_governance_read_test.go

```diff
@@ -18,7 +18,8 @@ func TestGetCommitteeInfo(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, epochId, committeeInfo.EpochId)
 	// just use a arbitrary big number to ensure there are enough validator
-	require.Greater(t, len(committeeInfo.Validators), 3)
+	// TODO: is this assertion needed? why not >= 1?
+	// require.Greater(t, len(committeeInfo.Validators), 3)
```

Comment:
I ran this test on my local side without this change. The one before this change passed the test

Reply:
did you do `docker pull iotaledger/iota-tools:devnet` ?

Reply:
Based on the iota implementation `crates/iota/src/iota_commands.rs:74`, I think we should set this to 
`require.Equal(t, len(committeeInfo.Validators), 1)`